NuRadioReco/examples/RNOG/rnog_standard_data_processing.py

```diff
@@ -71,11 +71,11 @@ def process_data(config):
     hardwareResponseIncorporator = NuRadioReco.modules.RNO_G.hardwareResponseIncorporator.hardwareResponseIncorporator()
     hardwareResponseIncorporator.begin()
 
-    channelBlockOffsetFitter = NuRadioReco.modules.RNO_G.channelBlockOffsetFitter.channelBlockOffsetFitter()
+    channelBlockOffsetFitter = NuRadioReco.modules.RNO_G.channelBlockOffsetFitter.channelBlockOffsets()
```

Comment:
autsch, again sorry that you had to fix the code, it is always a problem if you do not run it before. That being said I think we need to add this to the CI asap